CUDADataFormats/TrackingRecHit/src/classes_def.xml

```diff
@@ -3,14 +3,19 @@
   <class name="TrackingRecHitSoAHostPhase1" persistent="false"/>
   <class name="edm::Wrapper<TrackingRecHitSoAHostPhase1>" persistent="false"/>
 
+  <class name="TrackingRecHitSoAHostHIonPhase1" persistent="false"/>
+  <class name="edm::Wrapper<TrackingRecHitSoAHostHIonPhase1>" persistent="false"/>
+
   <class name="TrackingRecHitSoAHostPhase2" persistent="false"/>
   <class name="edm::Wrapper<TrackingRecHitSoAHostPhase2>" persistent="false"/>
 
-  <class name="TrackingRecHitSoADevicePhase1" persistent="false"/>
```

Comment:
are you sure we do not need this ?
I'm really asking, I did not check myself

Reply:
Assuming that the various Patatrack wfs running smoothly is enough as proof, apparently not. I can try to run some of them locally to double check.

Reply:
Ah, I see: `cms::cuda::Product<T>` declares the data member of type `T` to be transient, so ROOT does not need a dictionary for it.